src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -344,6 +344,13 @@ protected set
         // Functions with optional parameters have more than one signature.
         public abstract IEnumerable<TexlStrings.StringGetter[]> GetSignatures();
 
+        // Return all enums that are required by this function.
+        // This can be used to generate a list of enums required for a function library.
+        public virtual IEnumerable<string> GetRequiredEnumNames()
+        {
+            return new List<string>();
```

Comment:
>List<string>(); [](http://example.com/codeflow?start=23&length=15)

nit: 
This allocates a new List object on each time.  
0-length array would be better:  `new string[0]`
Or even  `Enumerable.Empty<string>();`